pkg/webhook/workspace/mutating_cfg.go

```diff
@@ -29,72 +29,111 @@ func buildMutateWebhookCfg() *v1beta1.MutatingWebhookConfiguration {
 	mutateWebhookPath := mutateWebhookPath
 	labelExistsOp := metav1.LabelSelectorOpExists
 	equivalentMatchPolicy := v1beta1.Equivalent
-	return &v1beta1.MutatingWebhookConfiguration{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: mutateWebhookCfgName,
+
+	workspaceMutateWebhook := v1beta1.MutatingWebhook{
+		Name:          "mutate.che-workspace-controller.svc",
+		FailurePolicy: &mutateWebhookFailurePolicy,
+		ClientConfig: v1beta1.WebhookClientConfig{
```

Comment:
Consider extracting client config to a separate variable.

Reply:
Done, good point.